client/landing/stepper/declarative-flow/internals/steps-repository/sensei-plan/styles.scss

```diff
@@ -156,34 +156,47 @@
 			color: $sensei-green-80;
 		}
 	}
+}
 
-	.plans-interval-toggle__popover {
-		&.plans-interval-toggle__popover--mobile {
-			margin-top: 10px;
+.plans-interval-toggle__popover {
+	.components-popover {
+		&__arrow::before {
+			background-color: $sensei-gray-100;
 		}
-
-		&:not(.plans-interval-toggle__popover--mobile) {
-			display: none;
-			@media ( min-width: 782px ) {
-				display: block;
-				margin-left: 15px;
-			}
+		&__triangle-bg {
+			fill: $sensei-gray-100;
 		}
 	}
 
-	.plans-interval-toggle--monthly .segmented-control {
-		margin-bottom: 32px;
-		@media (min-width: 782px) {
-			margin-bottom: 0;
-		}
+	&.plans-interval-toggle__popover--mobile {
+		margin-top: 10px;
+	}
+	&.plans-interval-toggle__popover--desktop {
+		margin-left: 15px;
 	}
 
-	.components-popover {
-		&__arrow::before {
-			background-color: $sensei-gray-100;
+	&.plans-interval-toggle__popover--mobile,
+	&.plans-interval-toggle__popover--desktop {
+		display: none !important;
+	}
+
+	@media ( min-width: 782px ) {
+		&.plans-interval-toggle__popover--desktop {
+			display: block !important;
 		}
-		&__triangle-bg {
-			fill: $sensei-gray-100;
+	}
+
+	@media ( max-width: 782px ) {
```

Comment:
Do you think converting this part to min-width and writing the bigger screen styles inside that, and making the mobile specific style the default would be better? Just to make it have a bit more mobile first approach

Reply:
Sure! Good point! Changed here: https://github.com/Automattic/wp-calypso/pull/84175/commits/fa62b0df8d692af77f4e0aebf8cf7c022ca37f8b

It got even cleaner! 🙌

Reply:
Mobile first FTW! 🥳